docs/content/configuration/csi_ug_config_create_vol_snapshotclass.md

```diff
@@ -17,18 +16,24 @@ When configuring the file, be sure to use the same array secret and array secret
     
```

Comment:
not relevant to current changes, but I couldn't but notice on line number 15 (on which I cannot comment directly):
```
the `CSI_` prefix is added as default
```
while technically true, to me this sounds like it implies that the default (svc) prefix parameter value is `CSI_`, while it's actually just `CSI` (the underscore is added for user-defined prefixes as well)

Reply:
@oriyarde  should I update this to just read `CSI`? It's not a problem

Reply:
that would be great, thanks

Reply:
DONE